packages/shortest/src/core/framework-detector/index.ts

```diff
@@ -85,3 +103,22 @@ export const detectFramework = async (options: { force?: boolean } = {}) => {
     throw new ShortestError("Failed to save project information");
   }
 };
+
+const detectNextJsDirPathFromConfig = async (): Promise<string | undefined> => {
+  const log = getLogger();
+  const paths = await getPaths(process.cwd());
+
+  const nextDirConfigPaths = paths
+    .filter((filePath) => /next\.config\.(js|ts|mjs|cjs)$/.test(filePath))
+    .map((filePath) => path.dirname(filePath));
+
+  if (nextDirConfigPaths.length > 0) {
+    log.trace("Detected Next.js config paths", { nextDirConfigPaths });
+    const nextNamedDir = nextDirConfigPaths.find((dirPath) =>
+      /next/i.test(dirPath),
+    );
+    return path.join(process.cwd(), nextNamedDir || nextDirConfigPaths[0]);
```

Comment:
If multiple Next.js apps are found within the same repo, prioritize the one that has `next` in the folder name (e.g. for Helper, will pick [`apps/next`](https://github.com/antiwork/helper/tree/7eeedad739d04f3f8d702bb88c2ab9832b79ec91/apps/nextjs) over [`apps/docs`](https://github.com/antiwork/helper/tree/7eeedad739d04f3f8d702bb88c2ab9832b79ec91/apps/docs))